.vitepress/theme/DovecotLayout.vue

```diff
@@ -2,12 +2,26 @@
 import DefaultTheme from 'vitepress/theme'
 import { useData } from 'vitepress'
 
-const { theme } = useData()
+const { site, theme } = useData()
 const { Layout } = DefaultTheme
+
+const re = new RegExp("^[/][0-9.]+[/]?$")
+const prod = re.test(site._value.base)
+const latest = site._value.themeConfig.dovecot.base_url + "/latest/"
+
 </script>
 
 <template>
  <Layout>
+  <template #doc-before v-if="!prod">
+    <div class="container custom-block warning large">
```

Comment:
Everywhere else, we are using 1 space per child level, not 2

Reply:
oh the indentation.